payments_service/solitude/transaction.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from payments_service import solitude
+from payments_service.solitude import constants
+
+log = logging.getLogger(__name__)
+
+
+class Transaction(object):
+    key = 'transaction_pk'
+
+    def __init__(self, session):
+        self.session = session
+        self.id = self.session.get(self.key, None)
+        self.api = solitude.api()
+
+    def create(self, user, plan_id, **kwargs):
+        """
+        Creates the session in solitude and stores the id in the session.
+        """
+        # For the moment, force a reset.
```

Comment:
could this comment be more specific? It doesn't make sense to me that you'd need to clear the session before creation.